src/validator.ts

```diff
@@ -248,6 +258,15 @@ export default function validator(node: Node, config: Config) {
         message: `Missing required attribute: '${key}'`,
       });
 
+  if (schema.slots)
+    for (const [key, { required }] of Object.entries(schema.slots))
+      if (required && node.slots[key] === undefined)
+        errors.push({
+          id: 'slot-missing-required',
+          level: 'error',
+          message: `Missing required slot: '${key}'`,
```

Comment:
Should we add a unit test for this string?

Reply:
There's a marktest case for this: https://github.com/markdoc/markdoc/pull/356/files#diff-e0ce4764821307e283506300a5110e8c134dad772d1cd61afce332d087832834R1729